### PR TITLE
[NUI] Document the new tags and attributes in Markup

### PR DESCRIPTION
**Pull request by shrouqsabah**
### Change Description ###

Document the new tags and attributes in Markup.
These tags and attributes have been added to DALi.
They are explained under current Markup documentation  "Use markup styling to style TextLabel".

For instance,
 - Issue #1618

### CODE REVIEW

**Comment by shrouqsabah** [comment]
Hello @wonrst ,
Please could you check it and let's me know what you see.

File: docs/application/dotnet/guides/user-interface/nui/text.md [hunk 1]
```diff
@@ -312,10 +312,169 @@ The following markup elements are currently supported:
...
+    The following example shows how to apply it:
+
+    ```xaml
+    <base:TextLabel x:Name="txt" EnableMarkup="True"
+        Text="<char-spacing value='5.0f'>Hello world</char-spacing>"/>
+    ```
+
+-   `<p>`
+
+    Use paragraph tag to set many styles on paragraph's level for the lines inside the element.
```
Comment (omar-srbd) [inline docs/application/dotnet/guides/user-interface/nui/text.md:404]: Missing article before the word "paragraph"
```suggestion
    Use paragraph tag to set many styles on a paragraph's level for the lines inside the element.
```

File: docs/application/dotnet/guides/user-interface/nui/text.md [hunk 2]
```diff
@@ -312,10 +312,169 @@ The following markup elements are currently supported:
...
+
+    The following attributes are supported:
+
+    -   `align`: Use the `align` attribute to define the horizontal alignment of paragraph. The supported attribute values are `begin`, `center` and `end` .
```
Comment (omar-srbd) [inline docs/application/dotnet/guides/user-interface/nui/text.md:408]: Missing article before the word "paragraph"
```suggestion
    -   `align`: Use the `align` attribute to define the horizontal alignment of a paragraph. The supported attribute values are `begin`, `center` and `end` .
```

File: docs/application/dotnet/guides/user-interface/nui/text.md [hunk 3]
```diff
@@ -312,10 +312,169 @@ The following markup elements are currently supported:
...
+        Text="<p>line 1</p><p rel-line-height=0.5>line\n2</p>line 3<p rel-line-height=3>line\n4</p>line 5"/>
+    ```
+
+-   `<item>`
+
+    Defines an embedded item within the text.
+
+    The following attributes are supported:
+
+    -   `url`: The url path of the image.
```
Comment (omar-srbd) [inline docs/application/dotnet/guides/user-interface/nui/text.md:431]: URL should be capitalized here. Please look into it.
```suggestion
    -   `url`: The URL path of the image.
```

File: docs/application/dotnet/guides/user-interface/nui/text.md [hunk 4]
```diff
@@ -312,10 +312,169 @@ The following markup elements are currently supported:
...
+    -   `color-blending`: Use the color-blending attribute to define whether the color of the image is multiplied by the color of the text.
+
+    The following example shows how to apply it:
+
+    ```xaml
+    <base:TextLabel x:Name="txt" EnableMarkup="True"
+        Text="<item 'width'=26 'height'=26 'url'='path/image.png' 'color-blending'='multiply'/>"/>
+    ```
+    > [!NOTE]
+    > The url of the image is optional. If there is no image the layout engine will use the width and height to create a space inside the text. This gap can be filled later.
```
Comment (omar-srbd) [inline docs/application/dotnet/guides/user-interface/nui/text.md:443]: URL should be capitalized here. Please look into it.
```suggestion
    > The URL of the image is optional. If there is no image the layout engine will use the width and height to create a space inside the text. This gap can be filled later.
```

**Review by omar-srbd — COMMENTED** [review]
Hello @shrouqsabah, I hope you are doing well.
Apart from the suggestions that I have made on the file in this PR. There were some instances where I could not comment.
Listing the issues below:
1.	Line 7: “…three Text components…”
Suggestion: “…three text components…”
2.	Line 24: “…and do not respond to…”
Suggestion: “…and does not respond to…”
Reason: Grammatical error.
3.	Line 242: “…`EnableAutoScroll` property to `true`:”
Suggestion: “…`EnableAutoScroll` property to `True`:”
Reason: Change has been suggested to maintain consistency with code snippet.
4.	Line 276: “…the `EnableMarkup` property of the `TextLabel` class to `true`:”
Suggestion: “…the `EnableMarkup` property of the `TextLabel` class to `True`:”
Reason: Change has been suggested to maintain consistency with code snippet.

File: docs/application/dotnet/guides/user-interface/nui/text.md (hunk 2 above)
Comment (shrouqsabah) [reply docs/application/dotnet/guides/user-interface/nui/text.md]: Done.

File: docs/application/dotnet/guides/user-interface/nui/text.md (hunk 3 above)
Comment (shrouqsabah) [reply docs/application/dotnet/guides/user-interface/nui/text.md]: Done

File: docs/application/dotnet/guides/user-interface/nui/text.md (hunk 4 above)
Comment (shrouqsabah) [reply docs/application/dotnet/guides/user-interface/nui/text.md]: Done

File: docs/application/dotnet/guides/user-interface/nui/text.md (hunk 1 above)
Comment (shrouqsabah) [reply docs/application/dotnet/guides/user-interface/nui/text.md]: Done

**Comment by shrouqsabah** [comment]
> Hello @shrouqsabah, I hope you are doing well. Apart from the suggestions that I have made on the file in this PR. There were some instances where I could not comment. Listing the issues below:
> 
>     1. Line 7: “…three Text components…”
>        Suggestion: “…three text components…”
> 
>     2. Line 24: “…and do not respond to…”
>        Suggestion: “…and does not respond to…”
>        Reason: Grammatical error.
> 
>     3. Line 242: “…`EnableAutoScroll` property to `true`:”
>        Suggestion: “…`EnableAutoScroll` property to `True`:”
>        Reason: Change has been suggested to maintain consistency with code snippet.
> 
>     4. Line 276: “…the `EnableMarkup` property of the `TextLabel` class to `true`:”
>        Suggestion: “…the `EnableMarkup` property of the `TextLabel` class to `True`:”
>        Reason: Change has been suggested to maintain consistency with code snippet.

Hello @omar-srbd , I hope you are doing well too.
I applied your suggestions, would you check them please.
Thanks,

**Review by omar-srbd — APPROVED** [review]
@shrouqsabah, thanks for incorporating the changes.

@mijin85cho @asm-foysal I am approving this PR for merging.
Please proceed with the merging.
Thank you.

**Pull request merged**